src/FluentlyHttpClient/FluentHttpResponse.cs

```diff
@@ -111,4 +111,16 @@ public FluentHttpResponse(HttpResponseMessage message, IDictionary<object, objec
 			Items = items ?? new Dictionary<object, object>();
 		}
 	}
+
+	/// <summary>
+	/// Fluent HTTP response, which wraps the <see cref="HttpResponseMessage"/> and adds GraphQL data
+	/// </summary>
+	/// <typeparam name="T"></typeparam>
+	public class GqlResponse<T>
```

Comment:
Since a GQL response only has a `Data` property and nothing extra, it would be nicer if somehow we unwrap it and give the data directly in the response

e.g.

currently, if you have the following response

```json
"data": {
   "person": {
        "name": "yordle"
    }
}
```

```cs
// you need to access the name as following:
request.Data.Data.Person.Name

// ideally it will be come
request.Data.Person.Name
```

This should be possible with the following: 
Modify `ReturnAsGqlResponse`
 - Change return type from `Task<FluentHttpResponse<GqlResponse<T>>>` => `Task<FluentHttpResponse<T>>
 - Parse with `builder.ReturnAsResponse<GqlResponse<T>>().` 
 - Read data from `.Data`  and return a `new FluentHttpResponse<T>`